sc-platform-api/src/documentation/article/create.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiDocumentationParams } from '../decorators';
+import { DefaultUnauthorizedResponse } from '../default/unauthorized.response';
+
+export const Create: ApiDocumentationParams = {
```

Comment:
Follow the naming style
```suggestion
export const ArticleDocumentationCreate: ApiDocumentationParams = {
```